lib/tmdb/networks/networks.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Tmdb.Networks do
+  use HTTPoison.Base
+
+  @doc ~S"""
+  Find networks by ID
+    ## Required Parameters
+    api_key
+    ## Optional Parameters
+    append_to_response:	Comma separated, any company method
+
+    ## Examples
+
+    iex> Tmdb.Companies.find(420)
+    %{"description" => nil, "headquarters" => nil, "homepage" => nil, "id" => 420,
+      "logo_path" => "/x1ynrDnk4RgbsnMI9yrdcdEm6Fu.png", "name" => "Marvel Studios",
+      "parent_company" => nil}
+ """
+  def find(id) do
+    get!("network/#{id}?").body
+  end
+
+  def tv(id) do
```

Comment:
Networks.tv/1 calls the `discover` api. It should probably be `Tmdb.Discover.tv/1` where the argument is a map to match the docs: http://docs.themoviedb.apiary.io/#reference/discover